src/transforms/coerce.jl

```diff
@@ -0,0 +1,34 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Coerce(pairs, tight=false, verbosity=1)
+
+Return a copy of the table, ensuring that the element scitypes of the columns match the new specification.
+
+Valid specifications (more to be added):
+
+(1) one or more column_name=>Scitype pairs
+"""
+struct Coerce{P, T, V} <: Transform
+  pairs::P
+  tight::T
+  verbosity::V
```

Comment:
tight is always a Bool? verbosity is always an Int? Why can't we fix these types in the struct? Type parameters come with additional compilation time.